### PR TITLE
[DA-61]: Implement the enpoint /reports/gav

### PR DESCRIPTION
**Pull request by jbartece**

### CODE REVIEW

**Comment by jbartece** [comment]
retest this please

**Comment by jomrazek** [comment]
:+1:

File: reports-rest/src/main/java/org/jboss/da/rest/reports/Reports.java [hunk 1]
```diff
@@ -48,20 +52,22 @@
...
     @Consumes(MediaType.APPLICATION_JSON)
     @Produces(MediaType.APPLICATION_JSON)
-    @ApiOperation(value = "Get dependency report for a GAV")
-    public Report gavGenerator(GAVRequest gavRequest) {
-        // TODO Auto-generated method stub
-        return null;
+    @ApiOperation(
+            value = "Get dependency report for a GAV " // TODO change when dependencies will be implemented
+                    + "(Currently the dependencies and dependency_versions_satisfied don't contains usefull values)")
+    public Report gavGenerator(GAV gavRequest) {
```
Comment (jomrazek) [inline reports-rest/src/main/java/org/jboss/da/rest/reports/Reports.java]: I think we should use GAVRequest not GAV here, typo @jbartece ?

Comment (jbartece) [inline reports-rest/src/main/java/org/jboss/da/rest/reports/Reports.java]: Currently we do not support products in requests so I removed it. I'm not sure if it is the best approach

**Pull request merged**